src/gaussiannumbers/GaussianNumberTypes.jl

```diff
@@ -3,7 +3,8 @@
 struct ZZiRing <: Ring
 end
 
-const FlintZZi = ZZiRing()
+const ZZi = ZZiRing()
```

Comment:
Why do we need these globals at all?

Reply:
per request of @thofma in https://github.com/Nemocas/Nemo.jl/pull/1935#issuecomment-2459431082